sources/assets/Xenko.Core.Assets/PackageSessionPublicHelper.cs

```diff
@@ -64,8 +63,10 @@ private static void CheckMSBuildToolset()
             // Check that we can create a project
             using (var projectCollection = new Microsoft.Build.Evaluation.ProjectCollection())
             {
-                if (projectCollection.GetToolset("15.0") == null)
-                    throw new InvalidOperationException("Could not find MSBuild toolset 15.0");
+                if (!projectCollection.Toolsets.Any(x => new Version(x.ToolsVersion).Major >= 15))
+                {
+                    throw new InvalidOperationException("Could not find a supported MSBuild toolset version");
```

Comment:
Let's add detail about expected version (15.0 or later)

Reply:
Would this be good?
"Could not find a supported MSBuild toolset version (expected 15.0 or later)"

Reply:
Yes, thanks